### PR TITLE
create admin tabel

### PR DESCRIPTION
**Pull request by abdou059**

### CODE REVIEW

**Review by mai-mohammed — CHANGES_REQUESTED** [review]
add id column

File: src/models/admin.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    allowNull: false,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  usernmae: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  password: {
+    type: DataTypes.TEXT,
```
Comment (mai-mohammed) [inline src/models/admin.ts]: change password type from TEXT to STRING

File: src/models/admin.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import sequelize from '../config/connection';
+
+const Admin = sequelize.define('admin', {
+  id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  usernmae: {
```
Comment (LinaYahya) [inline src/models/admin.ts]:
```suggestion
  username: {
```

File: src/models/admin.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+
+import sequelize from '../config/connection';
+
+const Admin = sequelize.define('admin', {
+  id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
```
Comment (LinaYahya) [inline src/models/admin.ts]: no need to allownull here as primary key guarantee that for you

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by tariqabusada — APPROVED** [review]

**Review by husamkamal — APPROVED** [review]

**Pull request merged**